### PR TITLE
Add `no_op_attribute` macro to `cairo-lang-macro-attributes`

### PR DESCRIPTION
**Pull request by DelevoXDG**
### Stack
- #1736 
- #1737 
- #1738

### CODE REVIEW

File: plugins/cairo-lang-macro/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+- Add `no_op_attribute` macro to `cairo-lang-macro-attributes` crate.
```
Comment (maciektr) [inline plugins/cairo-lang-macro/CHANGELOG.md:7]:
```suggestion
- Add `no_op_attribute` macro.
```

File: plugins/cairo-lang-macro-attributes/src/lib.rs [hunk 2]
```diff
@@ -170,3 +170,30 @@ pub fn executable_attribute(input: TokenStream) -> TokenStream {
     };
     TokenStream::from(expanded)
 }
+
+const NO_OP_ATTR_PREFIX: &str = "__no_op_attr_";
+
+#[proc_macro]
+pub fn no_op_attribute(input: TokenStream) -> TokenStream {
```
Comment (maciektr) [inline plugins/cairo-lang-macro-attributes/src/lib.rs:177]: Maybe we could create some common helper for this and `executable_attribute`, as they only differ by prefix?

**Review by maciektr — CHANGES_REQUESTED** [review]

File: plugins/cairo-lang-macro-attributes/src/lib.rs (hunk 2 above)
Comment (DelevoXDG) [reply plugins/cairo-lang-macro-attributes/src/lib.rs]: Sure 👍🏻

**Comment by DelevoXDG** [comment]
Discussed offline, will be addressed on the compiler side

**Pull request closed without merging**